src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Body.js

```diff
@@ -11,12 +11,17 @@ type Props = {
     /** @ignore */
     selectMode?: SelectMode,
     /** @ignore */
+    selectInFirstCell?: boolean,
+    /** @ignore */
     onRowSelectionChange?: (rowId: string | number, selected?: boolean) => void,
+    /** @ignore */
+    onRowToggleChange?: (rowId: string | number, expanded: boolean) => void,
```

Comment:
Why do we need a new callback?

Reply:
this is needed for toggling. to open and close the children. for the new arrow in the Row.js (onToggleChange)